h/static/scripts/vendor/jquery.scrollintoview.js

```diff
@@ -109,29 +109,37 @@
 
 				var animOptions = {};
 
+				var yDir = "none";
```

Comment:
> However, it's not only the "scrolling" which is problematic in those situations; the whole thing needs a full rethinking.

Exactly. I want to remove it all and find something better. I don't want something 80% in the meantime. I am advocating for the complete removal of the bucket bar and have for a long, long time.